sunkit_pyvista/plotter.py

```diff
@@ -279,7 +288,65 @@ def plot_field_lines(self, field_lines, **kwargs):
             grid = self._coords_to_xyz(field_line.coords.ravel())
             field_line_mesh = pv.StructuredGrid(grid[:, 0], grid[:, 1], grid[:, 2])
             color = {0: 'black', -1: 'tab:blue', 1: 'tab:red'}.get(field_line.polarity)
+            field_line_mesh.add_field_array([color], 'color')
             self.plotter.add_mesh(field_line_mesh, color=color, **kwargs)
             field_line_meshes.append(field_line_mesh)
 
         self._add_mesh_to_dict(block_name='field_lines', mesh=field_line_meshes)
+
+    def save(self, filepath, overwrite=False):
+        """
+        Adds all of the meshes in the current dictionary
+        to a :class:`~pyvista.core.MultiBlock`.
+
+        Parameters
+        ----------
+        filepath : `str` or `pathlib.Path`
+            Name of the file to save as, should have vtm or vtmb as an extension.
+
+        Notes
+        -----
+        This saves the rendered plot as a vtm extended file as well as a directory
+        of the individual meshes with the specified name.
+        """
+        file_path = Path(filepath)
+        directory_path = file_path.with_suffix('')
```

Comment:
@dstansby We need the file name without extension, not the parent directory, is there a better way to do this?

Reply:
To help me understand, can you give an example of what `filepath` could be, and what the resulting saved file and directory would be called?

Reply:
filepath = `/home/user/map_data.vtm`
in the /home/user/ directory a `map_data.vtm` file is stored as well as a directory named `map_data` in which each individual mesh of the multiblock is stored as a `.vtp` file.
directory_path is basically getting rid of the file extension to refer to the directory instead of the `.vtm` file

Reply:
Thanks - what you've done makes sense to me now :)